### PR TITLE
📝 Add deployment docs

### PR DESCRIPTION
**Pull request by rubenaeg**

### CODE REVIEW

**Comment by jankoenig** [comment]
Thanks. As discussed, it would be great to remove the table (just bullets with names and links to the respective docs?) and add a small section for vue deployment

**Comment by rubenaeg** [comment]
I updated the documentation accordingly, I hope this will be sufficient as a starting point.

File: README.md [hunk 1]
```diff
@@ -101,14 +101,22 @@ The `app.ts` file contains the initialization of the app and its plugins as well
...
-| [Google Cloud Functions](https://github.com/jovotech/jovo-framework/blob/master/docs/configuration/hosting/google-cloud-functions.md 'https://www.jovo.tech/docs/hosting/google-cloud-functions/google-cloud-functions') | Run the voice app on Google Cloud Functions |
-| [Azure Functions](https://github.com/jovotech/jovo-framework/blob/master/docs/configuration/hosting/azure-functions.md 'https://www.jovo.tech/docs/hosting/azure-functions')                                             | Run the voice app on Azure Functions        |
-| [Using nodejs built-in http server](https://github.com/jovotech/jovo-framework/blob/master/docs/configuration/hosting/http-host.md 'https://www.jovo.tech/docs/hosting/http-host')                                       | Run your app (almost) anywhere              |
+- [AWS Lambda](https://github.com/jovotech/jovo-framework/blob/master/docs/configuration/hosting/aws-lambda.md './https://www.jovo.tech/docs/hosting/aws-lambda')
```
Comment (jankoenig) [inline README.md:116]: I think this links will break. Please link to the official docs, not GitHub https://www.jovo.tech/docs/hosting/aws-lambda. etc.

File: README.md [hunk 2]
```diff
@@ -101,14 +101,22 @@ The `app.ts` file contains the initialization of the app and its plugins as well
 
 ## Deployment
 
+### Client
+
+If you are new to Vue.js and want to develop your website starting with this project, you can take a look at the Vue.js guide [here](https://vuejs.org/v2/guide/). This will give you a quick start to how Vue.js works, so you can start creating right away.
```
Comment (jankoenig) [inline README.md:106]: Would the general Vue.js info better fit into the "Customization" section?

File: README.md [hunk 3]
```diff
@@ -101,14 +101,22 @@ The `app.ts` file contains the initialization of the app and its plugins as well
...
+
+However, if you already have your own website and want to integrate the Jovo Chat Widget into your existing project, you can use the npm script `build` inside of the `client/` directory. This will produce a bundle in a dedicated `dist/` directory, containing the whole Vue.js client with minified HTML, CSS and JavaScript, ready for dynamic integration into any website.
+
+> You can also take a look at the Vue deployment docs [here](https://cli.vuejs.org/guide/deployment).
```
Comment (jankoenig) [inline README.md:110]: "Take a look at the..." and then add this to the top of this section

File: README.md [hunk 4]
```diff
@@ -101,14 +101,22 @@ The `app.ts` file contains the initialization of the app and its plugins as well
 
 ## Deployment
 
+### Client
+
+If you are new to Vue.js and want to develop your website starting with this project, you can take a look at the Vue.js guide [here](https://vuejs.org/v2/guide/). This will give you a quick start to how Vue.js works, so you can start creating right away.
+
+However, if you already have your own website and want to integrate the Jovo Chat Widget into your existing project, you can use the npm script `build` inside of the `client/` directory. This will produce a bundle in a dedicated `dist/` directory, containing the whole Vue.js client with minified HTML, CSS and JavaScript, ready for dynamic integration into any website.
```
Comment (jankoenig) [inline README.md:108]: "To integrate the the Jovo Chat Widget into your existing project [...]"

File: README.md (hunk 2 above)
Comment (rubenaeg) [reply README.md:106]: With the link to the deployment section at the top, it may be sufficient, yes. I'll apply the changes and push again :+1:

**Comment by jankoenig** [comment]
Thanks! 🎉

**Pull request merged**